src/loops/simple_programs_with_repitition/hard/q8/MainTest.java

```diff
@@ -0,0 +1,54 @@
+package loops.simple_programs_with_repitition.hard.q8;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+    // Java
+    public Clause[] testSentence(){
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "HowCouldThisHappen";
+        return new Clause[]{
+                new StringLiteral("Enter a sentence:"),
+                new Optional(new StringLiteral(" ")),
+                new NewLine(),
+                new StringLiteral("Socially Distanced String: "),
```

Comment:
Add the Optional with the space here as well

Reply:
I don't know, the space is visible in the sample and it's good programming format to leave a space there.